keyboards/takashicompany/minizone/readme.md

```diff
@@ -0,0 +1,53 @@
+# miniZone
+
+![miniZone](https://github.com/takashicompany/minizone/blob/master/images/qmk.jpg?raw=true)
```

Comment:
Could you rehost this on an image hosting site, such as imgur?

Reply:
@drashna 

Thanks for the review.
In a previous Pull Request for another keyboard, the review passed with no problem with an image link on the Github
If there are no problems, we would like to continue as is.

Reply:
Yes, sir.
Thank you for your kind explanation.
URL has been changed.